projects/core/src/user/occ/user.service.ts

```diff
@@ -26,16 +25,22 @@ const PAYMENT_DETAILS_ENDPOINT = '/paymentdetails';
 
 @Injectable()
 export class OccUserService {
-  // some extending from baseservice is not working here...
   constructor(protected http: HttpClient, protected config: OccConfig) {}
 
-  public loadUser(userId: string): Observable<User> {
+  loadUser(userId: string): Observable<User> {
     const url = this.getUserEndpoint() + userId;
     return this.http
       .get<User>(url)
       .pipe(catchError((error: any) => throwError(error)));
   }
 
+  updateUserDetails(username: string, user: User): Observable<{}> {
+    const url = this.getUserEndpoint() + username;
+    return this.http
+      .patch(url, user)
+      .pipe(catchError(error => throwError(error)));
+  }
+
```

Comment:
I didn't see any function that contains `patch` in its name. It can be considered as an implementation detail and not relevant for the caller.
But we can discuss it of course.

Reply:
Since we don't seem to have a use case where we use the PUT version of the endpoint, lets leave the function name as-is.